examples/sample.conf

```diff
@@ -1,32 +1,65 @@
 # vim: ft=dosini
 
-site = ArchWiki
-
-debug = true
+[DEFAULT]
+#
+# Common options.
+#
```

Comment:
This comment is pretty useless. If you want to explain what the `[DEFAULT]` section means, do it in the comment before the section.

Reply:
Done.